httpmw/limit_per_user_test.go

```diff
@@ -0,0 +1,59 @@
+package httpmw
```

Comment:
why it's httpmw ? If it's corrent name why it's not httpmw_test  or  httpmw_test _... ?

Reply:
We are not using test suffix in other tests as well, so I suggest we omit it here for consistency.